config/prod.exs

```diff
@@ -3,6 +3,15 @@ use Mix.Config
 # Do not print debug messages in production
 config :logger, level: :info
 
+# Networking module configuration:
+# load_from_system_env - false if not defined
+# ip_provider options: Uniris.Networking.IPLookup.Static, Uniris.Networking.IPLookup.Ipify, 
+# hostname - provides a constant IP address for Static
+# port - provides a P2P port number
+config :uniris, Uniris.Networking, 
+  load_from_system_env: true,
```

Comment:
This may be replaced by the runtime.exs generated during the release

Reply:
ok